packages/core/src/components/button/_common.scss

```diff
@@ -178,12 +172,9 @@ $button-intents: (
   }
 
   &:hover {
-    @include linear-gradient-with-fallback(
-      rgba($white, 0.1),
-      $white-transparent,
-      $hover-color
-    );
     box-shadow: $button-intent-box-shadow;
+    background-color: $hover-color;
+    background-image: $button-intent-gradient;
```

Comment:
do you need to repeat same `background-image` in pseudo states like `:hover` (compare to line 165)? 
in devtools, it seems safe to remove.

this comment applies several times throughout the changed files, i'm just gonna leave this one comment for them all.